package.json

```diff
@@ -25,6 +26,8 @@
     "github-cache": "^1.1.0",
     "hydrolysis": "^1.19.3",
     "nodegit": "^0.5.0",
+    "nomnom": "^1.8.1",
```

Comment:
Ha, oh god this is a nightmare. I think it was less than a month ago that I migrated polyup from yargs to nomnom.

Used the new hotness command-line-args, which polylint is already using. It seems nice.